scraper/main.py

```diff
@@ -1,2 +1,47 @@
-if __name__ == "__main__":
-    print("Hello World")
+import requests
+from bs4 import BeautifulSoup
+
+# URLs for FAANG internship listings
+urls = {
+    'Facebook': 'https://www.metacareers.com/jobs/?is_leadership=0&is_in_page=0',
+    'Amazon': 'https://www.amazon.jobs/en/search?base_query=internship',
+    'Apple': 'https://jobs.apple.com/en-ca/search?team=internships-STDNT-INTRN',
```

Comment:
https://github.com/BobaTalks/internship-tracker/pull/104#pullrequestreview-1407023905

Same comment on this url

Reply:
Sorry, the link should be https://jobs.apple.com/en-ca/search?location=canada-CANC+united-states-USA+mexico-MEXC&team=internships-STDNT-INTRN. I'm not sure how to change it, should I make another pull request?

Reply:
@lilyxmeng with pull requests on GitHub, you should be set up so your branch is tracked to this pull request. So you can make the change and commit to your repository and the pull request will update.